postman/collections/Interledger.json

```diff
@@ -3501,7 +3502,7 @@
 		{
 			"listen": "prerequest",
 			"script": {
-				"id": "83ff5ca5-119d-4e56-ba79-d7e8dc9aa1f8",
+				"id": "ca1da280-c4d6-43cc-86db-e3cd53bae676",
```

Comment:
not sure why all of these id's have been updated, but don't seem to affect anything 🤷

Reply:
I noticed with environment variables that the id's change if the order changes. It happened when I inserted a new variable in between existing, and they did not change when I added a new variable at the end.

I'm not sure if there is a similar thing happening here but I've noticed the same thing (bunch of id changes) on all of my postman changes.